src/node/server/serverPluginModuleRewrite.ts

```diff
@@ -137,13 +146,17 @@ export function rewriteImports(
       )
     }
 
+    const hasDefined = Object.keys(replacements).some((replacement) =>
```

Comment:
`hasDefined` is a bit unclear here, not sure about the proper naming here, maybe `needReplacements`?

Reply:
Thanks, that name is more clear.